spec/services/converters_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Converters do
+  subject { described_class }
```

Comment:
Better to just use `described_class` throughout the spec instead of `subject`. Usually `subject` is an instance of `described_class`.